src/Pass/IframeYouTubeTagTransformPass.php

```diff
@@ -114,7 +124,36 @@ protected function isYouTubeIframe(DOMQuery $el)
      * @param DOMQuery $el
      * @return string
      */
-    protected function getYouTubeCode(DOMQuery $el)
+    protected function getYouTubeCode(DOMQuery $el) {
+        $youtube_code = $this->_getYouTubeCode($el);
+
+        // Remove hash-tag timestamp jump:  #t=243 if present
+        $youtube_code = explode('#', $youtube_code);
+        return $youtube_code[0];
+    }
+
+    /**
+     *
+     * Get the youtube anchor hash (if available)
+     *
+     * @param DOMQuery $el
+     * @return string
+     */
+    protected function getYouTubeCodeHash(DOMQuery $el) {
+        $youtube_code = $this->_getYouTubeCode($el);
+        // Return hash-tag timestamp jump:  #t=243
+        $youtube_code = explode('#', $youtube_code);
+        return $youtube_code[1] ?? NULL;
```

Comment:
We couldn't wait for the Lullabot team to wake up (I tried sending them an e-mail but they're busy..), so we forked them and started fixing bugs in another repo.
https://github.com/GuideToIceland/amp-library

You can use that to get better AMP conversion code.